<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Look in:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="path_nav_cbbox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="go_up_btn">
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="list_radio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>list</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="detail_radio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>detail</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
     <item>
      <widget class="QListWidget" name="side_bar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="main_view_hbox" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QListView" name="list_view">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="flow">
          <enum>QListView::TopToBottom</enum>
         </property>
         <property name="isWrapping" stdset="0">
          <bool>true</bool>
         </property>
         <property name="layoutMode">
          <enum>QListView::SinglePass</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="tree_widget">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Size</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Date Modified</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="filename_edit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QCheckBox" name="sequences_ckb">
           <property name="styleSheet">
            <string notr="true">margin-right: 10px;</string>
           </property>
           <property name="text">
            <string>sequences</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Files of type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="files_type_cbbox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dialog_btnbox">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
